### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by rishabh486**
Screenshot (49)
Screenshot (50)
pr cretaed for card component

### CODE REVIEW

File: cards.css [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-grow: 1;
+}
+.top-3 {
+  display: flex;
+  flex-direction: row;
```
Comment (Muzammil-khan710) [inline cards.css:20]: Suggestion : by default flex direction is row

File: cards.css [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  justify-content: space-evenly;
+  align-items: center;
+
+  margin-bottom: 0.8rem;
```
Comment (Muzammil-khan710) [inline cards.css:23]: Remove extra spaces

File: cards.css [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+.top-1 p {
+  margin-left: 1.4rem;
+  margin-top: -2px;
+  line-height: 2rem;
+  width: 20rem;
+  font-size: 1.8rem;
+}
+.top-2 p {
+  margin-left: 1rem;
+
```
Comment (Muzammil-khan710) [inline cards.css]: Extra space

**Review by Muzammil-khan710 — APPROVED** [review]
Maintain good spacing in css file. Approving as of now

File: cards.css (hunk 3 above)
Comment (rishabh486) [reply cards.css]: removed extra spaces

File: cards.css (hunk 2 above)
Comment (rishabh486) [reply cards.css:23]: removed

File: cards.css [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+}
+.card-margin {
+  margin: 2rem;
+}
+.card-horizontal {
+  display: flex;
+  flex-direction: row;
+  width: 55rem;
+  max-height: 31rem;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
```
Comment (srishtigautam21) [inline cards.css:94]: use rem here

**Review by srishtigautam21 — APPROVED** [review]

**Pull request merged**